### PR TITLE
Add new exporting "home base" document

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

This PR creates a new "home base" for the exporting engine at `/docs/exporting/README.md` to let the original file at `/exporting/README.md` serve as a reference guide with the complete configuration options, and so on.

The goal here is to create a shorter document to help onboard users to exporting and/or get the migrated from backends quickly.

##### Component Name

area/exporting
area/docs

### CODE REVIEW

File: docs/export/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+Next, edit and uncomment any other lines necessary to connect the exporting engine to your endpoint. You must change the
+`destination` setting in most situations.
+
+For details on all the configuration options, see the [exporting reference](/exporting/README.md#configuration).
+
+> When migrating from backends, keep in kind that most settings for the exporting engine are identical to their backend
+> counterpart. You should be able to copy your backends settings into `exporting.conf` to complete your migration. The
+> only setting that won't work is `host tags`&mdash;use [`host labels`](/docs/tutorials/using-host-labels.md) instead.
```
Comment (thiagoftsm) [inline docs/export/README.md:86]: @joelhans I think there is a problem with the text "`host tags`&mdash;use", we do not have spaces between the values and I do not see a specific reason to use `&mdash;`.

File: docs/export/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Let's cover the process of enabling an exporting connector, using the Graphite connector as an example. These steps can
+be applied to other connectors as well.
+
+> If you are migrating from the deprecated backends system, this quickstart will also help you update your configuration
+> to the new format.
+
+Open the `exporting.conf` file with `edit-config`.
+
+```bash
+cd /etc/netdata   # Replace this path with your Netdata config directory, if different as found in the steps above
```
Comment (vlvkobal) [inline docs/export/README.md:31]: "as found in the steps above" - which steps? It looks like this part should be removed.

File: docs/export/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    # send charts matching = *
+    # send hosts matching = localhost *
+```
+
+Next, edit and uncomment any other lines necessary to connect the exporting engine to your endpoint. You must change the
+`destination` setting in most situations.
+
+For details on all the configuration options, see the [exporting reference](/exporting/README.md#configuration).
+
+> When migrating from backends, keep in kind that most settings for the exporting engine are identical to their backend
```
Comment (vlvkobal) [inline docs/export/README.md:84]: kind -> mind

File: docs/export/README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+> counterpart. You should be able to copy your backends settings into `exporting.conf` to complete your migration. The
+> only setting that won't work is `host tags`. Instead, use [host labels](/docs/tutorials/using-host-labels.md) to tag
```
Comment (vlvkobal) [inline docs/export/README.md:86]: Should we mention here anything about the `type` option and section names, like in https://github.com/joelhans/netdata/blob/exporting/exporting/README.md#L24?

Comment (joelhans) [reply docs/export/README.md]: Good call. I added a mention earlier in the document.

File: docs/export/README.md (hunk 2 above)
Comment (joelhans) [reply docs/export/README.md]: Good catch.

**Comment by joelhans** [comment]
Thanks, gentlemen!

**Pull request merged**